engine/installation/linux/docker-ce/debian.md

```diff
@@ -31,7 +31,7 @@ Raspbian versions:
 - Jessie 8.0 (LTS) / Raspbian Jessie
 - Wheezy 7.7 (LTS)
 
-Docker CE is supported on both `x86_64` and `armhf` architectures for Jessie and
+Docker CE is supported on both `amd64` (`x86_64`) and `armhf` architectures for Jessie and
```

Comment:
I think this is backward from what we wanted. :( I think it should be "`x86_64` (including `amd64`)" or just listing `x86_64` and `amd64` separately, but I prefer the first way. This feedback throughout.